app/scripts/metamask-controller.js

```diff
@@ -1283,6 +1283,21 @@ export default class MetamaskController extends EventEmitter {
       },
     );
 
+    this.controllerMessenger.subscribe(
+      `${this.snapController.name}:snapUpdated`,
```

Comment:
This is not compatible with the current version of skunkworks and needs to wait until we cut that release and include it in the extension.

Reply:
ok, will keep this on standby.